resources/views/categories/index.blade.php

```diff
@@ -0,0 +1,68 @@
+@extends('layouts.master')
+
+@section('main-content')
+    <div class="box">
+        <div class="box-header">
+            <h3 class="box-title">List Categories</h3>
+            <button id="btn-add-category" class="btn btn-primary" href="{{ route('categories.create') }}" style="float: right" title="Add Category">
+                <span class="glyphicon glyphicon-plus"></span>
+            </button>
+        </div>
+        <!-- /.box-header -->
+        <div class="box-body">
+            <div id="list-category-wrapper" class="dataTables_wrapper form-inline dt-bootstrap">
+                <div class="row">
+                    <div class="col-sm-12">
+                        <table id="list-category" class="table table-bordered table-striped dataTable table-hover" role="grid"
+                               aria-describedby="list-category-info">
+                            <thead>
+                            <tr role="row">
+                                <th style="width: 1em">ID</th>
+                                <th style="width: 7em">Name</th>
+                                <th>Description</th>
+                                <th style="width: 8em">Edit</th>
+                                <th style="width: 1em">Delele</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            @foreach ($categories as $category)
+                                <tr>
+                                    <td>{{ $category->id }}</td>
+                                    <td>{{ $category->name  }}</td>
+                                    <td>{{ $category->description }}</td>
+                                    <td>
+                                        <a class="btn btn-xs btn-success btn-edit-category" href="{{ route('categories.edit', ['id' => $category->id])}}">
```

Comment:
`href="{{ route('categories.edit', ['id' => $category->id])}}"` this is ok but there is another shorter way
`href="{{ route('categories.edit', $category->id)}}"`